covidfaq/evaluating/top_k_evaluator.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
```

Comment:
we can just add an option in evaluator.py to peform top-k as well. so we can avoid duplicating the script - and we can run all in one shot..

Reply:
Done